apps/api/src/domain/games/memory-match/queries/__snapshots__/memory-match-queries.fetch-by-id.e2e.spec.ts.snap

```diff
@@ -1,12 +1,271 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`when querying for a memory match round: fetch by Id when the user is authenticated as a coscrad admin when the memory match round is private should return the expected result 1`] = `
```

Comment:
There should be one and only one snapshot per endpoint.

Reply:
Not sure how to fix this